### ISSUE TITLE
Preformatted formatting requirements

### ISSUE THREAD

**Issue by jrbowden** [daisy/ebraille#94]
The normal behaviour for words in a paragraph of text is that they can wrap to fill the available line width.
Sometimes, certain blocks of text must not be wrapped. Examples might include:
- particular mathematical equations
- systems of music in the bar-over-bar format
- any text which must appear unbroken on a line or lines

The standard CSS white-space: nowrap; can be used to accomplish this.
Note that it is likely that the HTML pre tag will be used.

**Comment by GeorgeKerscher** [comment]
I do not think you want to wrap the primary structure with pre, e.g., MathML. You may want to have the EOL in a pre.

**Comment by jrbowden** [comment]
Thus far, MathML is not in scope. The file stores braille characters.
I was thinking of cases such as say, matrix calculations, where the braille may well be deliberately set up in a particular way.
Of course, "ordinary" lines of mathematics may be broken at logical points, so no need for pre for things like y = mx + c.
Hope this clarifies...

**Comment by bertfrees** [comment]
Missing an example of this in the styling best practices document.
